pkg/metadata/metadata.go

```diff
@@ -22,6 +23,20 @@ func (m Metadata) ToStructPB() (*structpb.Struct, error) {
 	return structpb.NewStruct(newMap)
 }
 
+func (m Metadata) ToStringValueMap() (map[string]string, error) {
```

Comment:
Could use https://github.com/mitchellh/mapstructure for this

Reply:
removing this function also because metadata is no longer needed for activity log